app/service/merchant_service.rb

```diff
@@ -0,0 +1,22 @@
+class MerchantService
+  def self.all_merchants
+    response = parsed_response("/api/v1/merchants")
```

Comment:
(see also lines 7, 11)
We're `response` anywhere else in this method, so I don't think we need that local variable.